### PR TITLE
Add missing vietnamese translations

### LINKED ISSUE #38770: Missing translations for Vietnamese (vi)
Hello,

There are some missing translation for Vietnamese (vi). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.vi.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.vi.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.vi.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.vi.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by jschaedl**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #38770  
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.vi.xlf [hunk 1]
```diff
@@ -114,6 +114,26 @@
                 <source>Please provide a valid phone number.</source>
                 <target>Vui lòng cung cấp số điện thoại hợp lệ.</target>
             </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>Hộp kiểm có một giá trị không hợp lệ.</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Vui lòng nhập địa chỉ Email hợp lệ.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.vi.xlf]: Should it be "Email" or "email"?

**Review by Nyholm — COMMENTED** [review]
Thank you for this PR. 

I just have one small question.

File: src/Symfony/Component/Form/Resources/translations/validators.vi.xlf (hunk 1 above)
Comment (van2412) [reply src/Symfony/Component/Form/Resources/translations/validators.vi.xlf]: Hi, thank you for your question. "Email" is english. So it doesn't matter for Vietnamese to use "Email" or "email".

Comment (jschaedl) [inline src/Symfony/Component/Form/Resources/translations/validators.vi.xlf]:
```suggestion
                <target>Vui lòng nhập địa chỉ email hợp lệ.</target>
```

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.vi.xlf]: Okey. I didnt know that. 

Thank you

**Review by Nyholm — APPROVED** [review]
I am happy with this. 

👍 

Thank you

**Comment by derrabus** [comment]
Thank you @jschaedl and @van2412.

**Pull request merged**